spec/features/merchants/items/create_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'Merchant item create page' do
+  before :each do
+    @merchant = Merchant.create!(name: "Little Shop of Horrors")
+  end
+
+  it 'can create a new item as enabled' do
+    visit "/merchants/#{@merchant.id}/items/new"
```

Comment:
I'm assuming that the link to create the new item works from the index page? It's probably in the index spec?